### PR TITLE
Initial GUI updates for IML 6

### PR DESCRIPTION
**Pull request by jgrund**
Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: docs/Graphical_User_Interface_9_0.md [hunk 1]
```diff
@@ -1,877 +1,498 @@
...
+- **Error alerts**: Marked with red color, these messages indicate that the filesystem is down or severely degraded, that one or more filesystem components are currently unavailable. For example, both primary and secondary servers for a target are not running. An error often has a remedial action you can take by clicking the button.
 
-**Common Searches**
+Before you see an unresolved filesystem error or warning in the Activity view, this error or warning briefly pops up on the red bar across the page. Clicking **Details** on the bar will open the Status window and display the alerts.
```
Comment (jgrund) [inline docs/Graphical_User_Interface_9_0.md:422]: This red bar no longer exists. Instead we show browser level notifications on active alerts.

Comment (amenytykii) [reply docs/Graphical_User_Interface_9_0.md]: Replaced red bar alert references with browser-level notifications.

File: docs/Graphical_User_Interface_9_0.md [hunk 2]
```diff
@@ -1,877 +1,471 @@
...
+For any listed server, you can do:
 
-This chart is displayed for a selected MDT and shows the percentage of available files (inodes) used over time. Data is displayed for the specific metadata target selected. 
-![md_Graphics/File_Usage_Chart.png][f9.13]
+- **Reboot**: Initiate a server reboot.
+- **Shutdown**: Initiate an orderly server shutdown.
+  - Note that with both reboot or shutdown, until the server is back online, HA capable filesystem targets fail over to a peer server or, if they are non HA capable, become unavailable.
+- **Remove**: Remove the server, including dependent filesystem targets.
+- **Force Remove**: (only if the remove command was unsuccessful) Remove the server record from the configuration database without contacting the server. This also removes all dependent filesystem targets without any attempt to unconfigure them.
+  - Note that after a forced removal, you cannot add the server to another filesystem until removing the management software from the server. To do this, contact technical support.
```
Comment (johnsonw) [inline docs/Graphical_User_Interface_9_0.md:108]: I don't think we want to encourage them to contact support for something like this. I feel like we have instructions on how to clean this up don't we?

File: docs/Graphical_User_Interface_9_0.md [hunk 3]
```diff
@@ -1,877 +1,471 @@
...
 
-This chart is displayed for a selected OST and shows the percentage of metadata objects used over time. Data is displayed for the object storage target selected. 
-![md_Graphics/Object_Usage_Chart.png][f9.14]
+Alternatively, expand the list of volumes for a server on the left:
 
+<a id="f11.2.3"></a>
+![md_Graphics/server-volumes-alt.png][f11.2.3]
 
-**View this chart**
+If there are a lot of volumes, the list is paginated.
```
Comment (johnsonw) [inline docs/Graphical_User_Interface_9_0.md:136]:
```suggestion
If there are a lot of volumes, the list will be paginated.
```

**Comment by jgrund** [comment]
LGTM

**Pull request merged**